docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -69,6 +69,20 @@
 * Updated the Repository and Cart to display Sample Sheet and Metadata downloads as separate buttons for better visibility. <!--PEAR-2232-->
 * Minor text and styling improvements. <!--PEAR-452/2298/2341/2323/2253/2290-->
 
+* __Single-Cell RNAseq__:
+    * Visualize single-cell RNA-Seq data with cluster plots and gene expression overlays.
+* __OncoMatrix__:
+    * By default hide “splice region” consequence mutations.
```

Comment:
```suggestion
    * By default hide "splice region" consequence mutations.
```
I'll try to look for these, but remove any curly quotes that I don't catch.

Reply:
Thanks, fixed